content/blog/2025/04/2025-04-28-election-announcement.md

```diff
@@ -0,0 +1,97 @@
+
+If you believe you are eligible to participate but have not heard from us or [OpaVote](https://www.opavote.com/) – the election system we have chosen for this year’s elections – by May 24th, [please email us promptly](mailto:elections@foundation.matrix.org). We build our voter rolls on May 31st and can't change them after that!
+
+All of the candidates will be listed on an elections page here on our website which we’ll publish on May 17th.
+
+## How to prepare yourself
+
+If you want to be a candidate in the election, there are a few things you should know:
+
+* All candidates will be listed on our website, grouped by the constituency group.
+* We do not require people to use their “legal name.”
+* Candidates who are nominated by an organizational member will have their affiliation listed.
+* We expect candidates to provide a short bio and a short statement, collectively covering qualifications and priorities. Candidates may also provide a link to an extended statement or platform on their personal blog or website.
+* You are encouraged to make yourself available for conversation with members of your constituency group.
+* The period for campaigning runs from April 27th to May 17th. Please note our [community Code of Conduct](https://matrix.org/legal/code-of-conduct/) while campaigning.
+
+## What to expect if you win
+
+Congratulations! All of the winners will be announced on June 16th, but since the elections conclude on June 13th, you may get an email with the results of the election you participated in before June 16th.
+
+All elected representatives will be added to a private mailing list and Matrix room so that they can introduce themselves and communicate in between meetings. I will be reaching out to every elected representative to meet one-on-one, get acquainted, and answer questions before we convene for their first Governing Board meeting.
+
+The Governing Board meets as a full board twice a year, for at least 90 minutes each time, and meetings are facilitated by the duly elected Chair or Vice Chair. Expect one meeting before the end of the year, and another before the next election.
+
+You are likely to receive a packet of materials before each meeting which you will be expected to have reviewed. Most meetings will be held online, though we may convene meetings in-person, such as at The Matrix Conference. We will make every effort to make in-person meetings accessible to remote participants, and no official business will ever happen without appropriate notice, quorum, and minutes.
+
+While the Governing Board is an advisory board, members are encouraged to play a role in helping to carry out the activities that support the staff in delivering on the Foundation’s collective remit, such as coalescing priorities to share with the Spec Core Team and connecting with contributors and funders to implement proposals.
```

Comment:
Indeed before this election is over, we will *expect* members to contribute with a certain minimum effort. Even though this is not yet technically decided, should we be more upfront about this?

Reply:
Good thinking, I'd welcome a suggested form of words here – short of that, I think we can include that in our nomination materials when we post them